### DISCUSSION TITLE
SynthMorph - transfer learning

### DISCUSSION THREAD

**Discussion by id-b3** [voxelmorph/voxelmorph#435] [Q&A] [answered]
I read your SynthMorph paper with great interest, especially on the performance of the tool on non-target tasks by example of cardiac MRI. You achieve excellent results without additional training, however I was wondering if you tried additional transfer learning and whether that can improve results further? 
In my case, I'm interested in applying synthmorph on inspiratory/expiratory chest CT scans, and am decinding on the best way to start.
Would you recommend training VoxelMorph from scratch with paired CTs, or would SynthMorph +/- transfer learning be a viable alternative?

Thank you for such an impressive project!

Ivan

**Comment by mu40** [comment, marked as answer]
Hi Ivan, we did not experiment with transfer learning in conjunction with SynthMorph. I would try both VoxelMorph and initializing with the anatomy-agnostic SynthMorph (shapes) weights to see what works best. My guess would be that fine tuning the weights with CT-typical image contrast and larger and more global deformations will give you a boost in accuracy over the existing SynthMorph shapes weights (which we mostly evaluated on neuroimaging data).

**Reply by id-b3** [reply]
Thanks a lot for your advice, I'll try it out as you described!
